assignment1.txt

```diff
@@ -0,0 +1,9 @@
+Christopher Liu
+1. The return 0; at the end of the main() signifies that the function has completed with error 0, which is no error. Technically it is not needed for most compilers.
+2. the *r at the end should be /r as otherwise it will multiply instead of divide twice
+3. %d is for integers, for floats you would use %f
```

Comment:
I think this is your answer to number 2, but you're right about this being an error. There are a couple of more errors in this code - check if you can find them.